proposals/0282-atomics.md

```diff
@@ -258,12 +258,12 @@ Code that uses these unsafe atomic types must manually manage the lifecycle of t
 This is typically done by allocating a dynamic variable dedicated to holding storage for the atomic value:
 
 ```swift
-// Create an unsafe atomic integer, initialized to 0 
+// Create an initialized unsafe atomic value
 func atomicDemo<Value: AtomicProtocol>(initialValue: Value) {
   typealias Storage = UnsafeAtomic<Value>.Storage
```

Comment:
The comment only describes the first part of the function, so it could be moved:

```suggestion
func atomicDemo<Value: AtomicProtocol>(initialValue: Value) {
  // Create an initialized unsafe atomic value
  typealias Storage = UnsafeAtomic<Value>.Storage
```

Reply:
Fair.